docs/outlook/autolaunch.md

```diff
@@ -266,6 +266,17 @@ While implementing your own functionality, you may need to debug your code. For
 
 Runtime logging is also available for this feature on Windows. For more information, see [Debug your add-in with runtime logging](../testing/runtime-logging.md#runtime-logging-on-windows).
 
+## Deploy to users
+
+You can deploy event-based add-ins by uploading the manifest through the Microsoft 365 admin center. In the admin portal, expand the **Settings** section in the navigation pane then select **Integrated apps**. On the **Integrated apps** page, choose the **Upload custom apps** action.
+
+![Screenshot of the Integrated apps page on the Microsoft 365 admin center, including the Upload custom apps action](../images/outlook-deploy-event-based-add-ins.png)
+
+The option to deploy event-based add-ins to AppSource or update existing AppSource add-ins to include the event-based activation feature should be available soon.
+
+> [!IMPORTANT]
+> Event-based add-ins are restricted to admin-managed deployments only. For now, users can't get event-based add-ins from AppSource.
```

Comment:
@ElizabethSamuel-MSFT :  We need to call out that add-in can't be installed by end users from "AppSource and InClient Stores" 

Other aspects of the document looks good to go